docs/02_cleos/03_command-reference/push/push-action.md

```diff
@@ -31,3 +29,65 @@ Push a transaction with a single action
 `--max-net-usage` _UINT_ - set an upper limit on the net usage budget, in bytes, for the transaction (defaults to 0 which means no limit)
 
 `--delay-sec` _UINT_ - set the delay_sec seconds, defaults to 0s
+
+## Output
+
+For actions that return a value `cleos` will have an extra line which will be prefixed with `=>`.
+
+Below you can see three examples of pushing actions `rstring` and `ruint` which return the value passed as input parameter to them:
```

Comment:
here is the code of the actions:
```
[[eosio::action]]
std::string rstring(const std::string & str) { return str;   }

[[eosio::action]]
uint16_t    ruint(uint16_t i)                { return i;     }

[[eosio::action]]
tmp         rtmp()                           { return tmp{}; }
```
It might be useful to include it as well

Reply:
thanks, done.